GETTING_STARTED.md

```diff
@@ -0,0 +1,106 @@
+# Getting started
+
+Firstly, start the application by running `python3 main.py`.
+
+You will be shown the main window of the application (pictured below) which contains both a list of components and a 3D view visualising the components listed with their respective shape and position. 
```

Comment:
how about `...list of _components_ which comprise the beamline, and a..."
using italics in introduce "component" as terminology which you will continue to use?